Gems/ROS2/Code/Source/Lidar/LidarTemplate.h

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+#pragma once
+
+#include <AzCore/RTTI/RTTI.h>
+#include <AzCore/Serialization/SerializeContext.h>
+#include <AzCore/std/string/string.h>
+
+namespace ROS2
+{
+    //! Configuration reflecting a specific Lidar model.
+    //! This is meant to capture differences between different Lidars available on the market.
+    //! @note Current implementation is simplified. Rays in real lidars are often not uniformly
+    //! distributed among angular range, there is noise etc.
+    struct LidarTemplate
+    {
+    public:
+        AZ_TYPE_INFO(LidarTemplate, "{9E9EF583-733D-4450-BBA0-ADD4D1BEFBF2}");
+        static void Reflect(AZ::ReflectContext* context);
+
+        // TODO - implement more models. Add at least one realistic model.
```

Comment:
This TODO should be made into a Github issue: https://github.com/o3de/sig-core/blob/main/governance/Coding-Standards-and-Style-Guide.md#todo-comments